doc-src/source/data-structure.rst

```diff
@@ -42,22 +42,19 @@ Rotations and Coordinate Systems
 
 Coordinate systems (a.k.a. reference frames) specify the coordinate
 directions of vector data in the data object. The values in the list
-`dat.props['rotate_vars']` specifies the vectors that are rotated when
-changing between different coordinate systems. **You can also create
-`dat._rotate_*` functions to perform additional rotation operations. I
-need to document these!**
-
-The first dimension of these vectors are the coordinate
-systems. DOLfYN supports four primary coordinate systems:
+``dat.props['rotate_vars']`` specifies the vectors that are rotated
+when changing between different coordinate systems.  The first
+dimension of these vectors are their coordinate directions, which are
+defined by the following coordinate systems:
 
 - **BEAM**: this is the coordinate system of the 'along-beam'
   velocities. When a data object is in this coordinate system, only
-  the velocity data (i.e., the variables in `dat.props['rotate_vars']`
-  starting with '`vel'`) is in beam coordinates. Other vector
-  variables listed in `'rotate_vars'` (e.g., `dat.orient.AngRt`, for
-  instruments with IMUs) are in the INST frame. This coordinate system
-  is *not* ortho-normal. When the data object is in BEAM coordinates,
-  the first dimension of the velocity vectors are: [beam1, beam2,
+  the velocity data (i.e., the variables in
+  ``dat.props['rotate_vars']`` starting with ``'vel'``) is in beam
+  coordinates. Other vector variables listed in `'rotate_vars'` (e.g.,
```

Comment:
"Other vector variables listed in `'rotate_vars'` (e.g.,
  `dat.orient.AngRt`) are in the INST frame."
Clarify the point of this statement. Is it because in DOLfYN the data will never start in BEAM, so if you've converted from INST to BEAM, only the vel data has been converted to BEAM?

Reply:
Yes. Can you help me come up with language that is clearer? (For simplicity: just comment here with thoughts/ideas, rather than making changes yourself, committing, submitting PR, etc.).

Reply:
On second thought, we *could* just simplify this and rotate all data into the beam coordinate system. I don't know why anyone would ever want to know e.g., AngRt in beam-coords, but it would make the docs simpler.

Reply:
That seems simpler. Then add a note to clarify 
(1) binary data files from an ADV w/IMU will be in XYZ coordinates (link to the Motion Correction Pre-Deployment Requirements), so the data set will only be switched to BEAM for data cleaning purposes and then only the vel data will be used for cleaning (link to adv.clean.py). The other rotate_vars will also be rotated into BEAM but will be irrelevant to the cleaning steps.
(2) what needs to be said about ADV w/o IMU and ADCP files? Those files will only have rotate_vars that are in the vel category (right?), so the above caveat doesn't matter for those files. For the sake of clarity here (or in The Basics) mention whether raw files from those instruments can be read in if the binary data is in BEAM or whether the binary files must be in XYZ or ENU/ETU.

Reply:
So, as discussed, this plan didn't pan out because rotating 3-d vectors to 4-beam or 5-beam coordinates is fraught with challenges. I've updated the docs as you discussed, and created issues #52 and #53 to think about this more in the future.